src/amo/components/SearchResult.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import 'core/css/SearchResult.scss';
+
+export default class SearchResult extends React.Component {
+  static propTypes = {
+    result: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <li className="SearchResult">
+        <Link to={`/en-GB/firefox/addon/${result.slug}/`}
```

Comment:
Locale should be dynamic here, too. It would be really nice to have something like django's `reverse('addons.detail', result.slug)` but react-router doesn't some with anything out of the box :(